packages/formstr-app/src/utils/relayUtils.ts

```diff
@@ -0,0 +1,50 @@
+import { RelayStatus } from '../containers/CreateFormNew/providers/FormBuilder/typeDefs'; // Adjust path
+
+/**
+ * @param url The WebSocket URL of the relay.
+ * @param timeoutMs The timeout in milliseconds for the connection attempt.
+ * @returns A promise that resolves to the RelayStatus.
+ */
+export const checkRelayConnection = (
+  url: string,
+  timeoutMs: number = 3000
+): Promise<RelayStatus> => {
+  return new Promise((resolve) => {
+    let ws: WebSocket;
+    let timeoutId: NodeJS.Timeout | undefined;
+
+    try {
+      ws = new WebSocket(url);
```

Comment:
why are we using oiur own custome Websockets? we should try and connect it via nostr tools itself, this would get redundant and hard to manage soon